sway/input/seatop_down.c

```diff
@@ -74,7 +74,7 @@ static void handle_tablet_tool_motion(struct sway_seat *seat,
 
 static void handle_unref(struct sway_seat *seat, struct sway_container *con) {
 	struct seatop_down_event *e = seat->seatop_data;
-	if (e->con == con) {
+	if (!con || e->con == con) {
```

Comment:
I don't think `con` can ever be null here. The only caller is https://github.com/swaywm/sway/blob/acf946fe4c495b47c5a76230ba98f9b973bec96d/sway/tree/container.c#L1054

Reply:
A layer surface has no `con`.

Reply:
Ah, yeah. This is unref. Indeed, can't be null.